apps/video/index.html

```diff
@@ -152,18 +167,20 @@ <h1 id="video-title"></h1>
     <!-- This element is the template of each thumbnail item -->
     <div id="thumbnail-group-header" class="hidden">
       <!--
-        <li>
-          <div class="thumbnail-group-header"></div>
-          <ul class="thumbnail-group-container"></ul>
+        <li role="presentation">
```

Comment:
I think it would be helpful to include a comment regarding the "presentation" role and why it's being used here.